Doc/whatsnew/3.11.rst

```diff
@@ -99,6 +99,17 @@ Optimizations
   almost eliminated when no exception is raised.
   (Contributed by Mark Shannon in :issue:`40222`.)
 
+* Method calls with keywords are now up to 20% faster due to bytecode
```

Comment:
Do we have a pyperformance run for this? I only found your comment here:

https://github.com/python/cpython/pull/26014#issuecomment-836557143

but I don't see the full run with LTO/PGO. This is important as normally we report 20% faster based on the pyperformance run, not for individual microbenchs.

Reply:
I ran pyperformance with LTO+PGO and saw no change because pyperformance code barely uses method calls with keywords. That entry is taken from the whats new in python 3.7 https://docs.python.org/3/whatsnew/3.7.html#optimizations and is also the result of a microbench. Maybe I should just remove that line :(?

Recently I've been thinking that pyperformance doesn't really have enough object-oriented benchmarks. Maybe I can look into adding some.

Reply:
> That entry is taken from the whats new in python 3.7 https://docs.python.org/3/whatsnew/3.7.html#optimizations and is also the result of a microbench.

Maybe I am missing something, but we did a pyperformance run for that:

https://bugs.python.org/msg282622

> Maybe I should just remove that line :(?

No, but surely clarify that a bit better: mention that is on a microbenchmark.

> Recently I've been thinking that pyperformance doesn't really have enough object-oriented benchmarks. Maybe I can look into adding some.

That's a good idea. Meke sure to sync also with @vstinner

Reply:
> > That entry is taken from the whats new in python 3.7 https://docs.python.org/3/whatsnew/3.7.html#optimizations and is also the result of a microbench.
> 
> Maybe I am missing something, but we did a pyperformance run for that:
> 
> https://bugs.python.org/msg282622
> 

Gah, I just realised my message to you was misworded. What I meant was "That entry is taken from the whats new in python 3.7 https://docs.python.org/3/whatsnew/3.7.html#optimizations and the number I provided is the result of a microbench.". Sorry.